test/e2e/core.go

```diff
@@ -62,6 +63,8 @@ func CoreDump(dir string) {
 		cmds = append(cmds, []command{
 			{"cat /var/log/kubelet.log", "kubelet"},
 			{"cat /var/log/supervisor/supervisord.log", "supervisord"},
+			{"cat /var/log/supervisor/kubelet-stdout.log", "supervisord-kubelet-stdout"},
+			{"cat /var/log/supervisor/kubelet-stderr.log", "supervisord-kubelet-stderr"},
```

Comment:
Do we also want to gather `/var/log/docker.log` and `docker-stdout/stderr.log`? I think `/var/log/docker.log` will be helpful.